docs/source/customizations/coprocessor.mdx

```diff
@@ -61,8 +61,6 @@ At _every_ stage, the router waits for your coprocessor's response before it con
 
 If your coprocessor hooks into your router's `SubgraphService`, the router sends a separate coprocessor request _for each subgraph request in its query plan._ In other words, if your router needs to query three separate subgraphs to fully resolve a client operation, it sends three separate coprocessor requests. Each coprocessor request includes the [name](#servicename) and [URL](#uri) of the subgraph being queried.
 
-> This behavior differs from hooking into `RouterService`, which can trigger exactly one coprocessor request per stage.
```

Comment:
Now that `RouterService` can trigger more than one coprocessor request per response via defer, this callout becomes more misleading than informative